### PR TITLE
HIVE-24657: Make Beeline Logging Dependencies Explicit

### PR DESCRIPTION
**Pull request by belugabehr**

### CODE REVIEW

File: beeline/pom.xml [hunk 1]
```diff
@@ -105,25 +121,47 @@
       <artifactId>super-csv</artifactId>
       <version>${super-csv.version}</version>
     </dependency>
+    <dependency>
+      <groupId>org.apache.logging.log4j</groupId>
+      <artifactId>log4j-slf4j-impl</artifactId>
+    </dependency>
```
Comment (belugabehr) [inline beeline/pom.xml:127]: Add Log4j 2 here

**Comment by belugabehr** [comment]
@nrg4878 FYSA

**Review by nrg4878 — APPROVED** [review]
Looks good to me. +1

**Comment by belugabehr** [comment]
Eh.  It actually needs some work.

Since HiveServer and Beeline share a lib directory (if they are co-installed), I need to update the start scripts to deal with that.

**Pull request merged**